python/dgl/graphbolt/minibatch.py

```diff
@@ -242,6 +256,8 @@ def blocks(self):
                 original_column_node_ids is not None
             ), "Missing `original_column_node_ids` in sampled subgraph."
             if is_heterogeneous:
+                for v in subgraph.sampled_csc.values():
+                    cast_to_minimum_dtype(v)
```

Comment:
`v = cast_to_minimum_dtype(v)`? or omit the return value if update on input argument?

Reply:
Returning self was helpful in the below case to avoid an extra assignment.

Reply:
Otherwise, the modification is in-place so this code works.